### ISSUE TITLE
MemoryError in sklearn/cluster/_k_means_elkan.pyx:229

### ISSUE THREAD

**Issue by amir20** [amir20/clashstats#461]
## Error in ClashLeaders

**MemoryError** in **sklearn/cluster/_k_means_elkan.pyx:229**

[View on Bugsnag](https://app.bugsnag.com/clash-leaders/clashleaders/errors/5b83646f2b818b001ce05be8?event_id=5b83646f002dd87563280000&i=gh&m=ci)

## Stacktrace

    sklearn/cluster/_k_means_elkan.pyx:229 - sklearn.cluster._k_means_elkan.k_means_elkan
    clashleaders/clustering/kmeans.py:23 - train_model
    clashleaders/batch/similar_clan.py:19 - compute_similar_clans
    clashleaders/scheduler.py:160 - main

[View full stacktrace](https://app.bugsnag.com/clash-leaders/clashleaders/errors/5b83646f2b818b001ce05be8?event_id=5b83646f002dd87563280000&i=gh&m=ci)

*Created automatically via Bugsnag*
